ocm-container.sh

```diff
@@ -157,7 +164,7 @@ ${AWSFILEMOUNT} \
 ${SSH_AGENT_MOUNT} \
 ${OPS_UTILS_DIR_MOUNT} \
 ${SCRATCH_DIR_MOUNT} \
--v ${HOME}/.ssh/sockets:/root/.ssh/sockets \
+-v /root/.ssh/sockets \
```

Comment:
This isn't a valid volume mount.  I'm assuming you meant to use the `$SOCKET_MOUNT`  var?

Reply:
Ah whoops. I will fix that.